src/osas/core/interfaces.py

```diff
@@ -145,10 +145,10 @@ def apply(self, func, axis: int = 0) -> any:
         pass
 
     @abstractmethod
-    def save(self, file_handle) -> None:
+    def save(self, file_name) -> None:
```

Comment:
this was intentionally a file handle, not a file name. We used this to store the model into a datastream.